### ISSUE TITLE
Prettify the `tdp browse`

### ISSUE THREAD

**Issue by sergkudinov** [TOSIT-IO/tdp-lib#349]
The output of `tdp browse` is not very well looking now:

<img width="1680" alt="image" src="https://github.com/TOSIT-IO/tdp-lib/assets/23199590/0d712b64-e718-40c8-9fc7-9ec92efea471">

### Suggestions:

- add empty lines between blocks
- do we need Sources and Targets printed again in separate blocks?
- when Service Component logs is empty, it prints nothing. It is not obvious

**Comment by PaulFarault** [comment]
I agree that we should enhance this output. However, keep in mind that the content of the database - and consequently the output of the `tdp browse` command - will change after adding the _stale configuration_ and _hosts distinction_ features. Therefore, I recommend waiting for version 1.1 features to be ready before modifying the content of this command.

In the meantime, feel free to add blank lines if you wish. In my opinion, the first table isn't appropriate in its current form. The `tdp browse [id]` command will always display a single result for which `key: value` pairs are more suited than a one line table.

**Comment by sergkudinov** [comment]
I'll make it now.

> the first table isn't appropriate in its current form.

agree

@PaulFarault Do you know why we displayed `Sources` and `Targets` in a table form?

**Comment by PaulFarault** [comment]
> Do you know why we displayed Sources and Targets in a table form?

Probably sth legacy. In this context, `sources` and `targets` operations can be displayed as list of strings.
